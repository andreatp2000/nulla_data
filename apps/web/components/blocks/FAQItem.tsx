interface FAQItemProps {
  q: string
  a: string
}

export function FAQItem({ q, a }: FAQItemProps) {
  return (
    <details className="rounded border border-nd-surface p-4">
      <summary className="cursor-pointer font-medium">{q}</summary>
      <p className="mt-2 text-sm text-nd-muted">{a}</p>
    </details>
  )
}
