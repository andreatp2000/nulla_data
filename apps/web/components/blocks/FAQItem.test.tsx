import { render } from '@testing-library/react'
import '@testing-library/jest-dom'
import { FAQItem } from './FAQItem'

test('faq item snapshot', () => {
  const { container } = render(<FAQItem q="Q" a="A" />)
  expect(container).toMatchSnapshot()
})
