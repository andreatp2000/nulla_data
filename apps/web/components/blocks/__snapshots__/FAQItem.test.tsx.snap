// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`faq item snapshot 1`] = `
<div>
  <details
    class="rounded border border-nd-surface p-4"
  >
    <summary
      class="cursor-pointer font-medium"
    >
      Q
    </summary>
    <p
      class="mt-2 text-sm text-nd-muted"
    >
      A
    </p>
  </details>
</div>
`;
